sdk/log/provider_test.go

```diff
@@ -287,3 +288,18 @@ func TestLoggerProviderForceFlush(t *testing.T) {
 		assert.ErrorIs(t, p.ForceFlush(ctx), assert.AnError, "processor error not returned")
 	})
 }
+
+func BenchmarkLoggerProviderLogger(b *testing.B) {
+	p := NewLoggerProvider()
+	names := make([]string, b.N)
+	for i := 0; i < b.N; i++ {
+		names[i] = fmt.Sprintf("%d logger", i)
+	}
+
+	b.ResetTimer()
+	b.ReportAllocs()
+
+	for i := 0; i < b.N; i++ {
+		_ = p.Logger(names[i])
```

Comment:
Is the compiler optimizing this empty assignment away? Should it be stored to a var outside of the loop scope to ensure it isn't?

Reply:
Sure, I've done that. There's no difference to the results though.